deployment/execution.js

```diff
@@ -57,23 +57,38 @@ async function runCommands(allScripts, results, args, callback) {
     let logs = ""
     var index = 0;
     var index2 = 0;
```

Comment:
you should use better names especially there are so many counters now. Something like `forkIndex` and `networkIndex` are easier to understand

Reply:
Good call!